gtk4/src/accessible.rs

```diff
@@ -75,3 +81,232 @@ impl<O: IsA<Accessible>> AccessibleExtManual for O {
         }
     }
 }
+
+pub enum Property<'p> {
```

Comment:
Ideally those enums could use some documentation & be exposed in lib.rs;

Reply:
Also needs #[non_exhaustive] & probably implement stuff like Debug

Reply:
Is there an easy way to get the docs generator to copy the docs from `GtkAccessibleProperty`, etc? That would probably make the most sense.

Reply:
No, that's not possible yet. We didn't have such use case so far.